scripts/tracker.lua

```diff
@@ -55,11 +74,82 @@ function Tracker.newTracker(trackerType, trackerList)
         newTracker.userCanEnable = false
         newTracker.enabled = false
         newTracker.realtimeCamera = true
+    elseif trackerType == "cityblock" then
+        -- cityblock-specific data
+        newTracker.cityBlock = Tracker.cityBlock:new()
+        newTracker.minPos = { x = -5, y = -5 }
+        newTracker.maxPos = { x = 5, y = 5 }
```

Comment:
These are set/recalculated by `recalculateCityBlock()` (on the next line)

Reply:
This was a kludge to prevent a nil dereference in an edge-case. I've fixed it better.